components/compositing/compositor.rs

```diff
@@ -379,7 +379,7 @@ impl IOCompositor {
             pending_scroll_zoom_events: Vec::new(),
             composite_target,
             shutdown_state: ShutdownState::NotShuttingDown,
-            page_zoom: Scale::new(1.0),
+            page_zoom: Scale::new(1.2),
```

Comment:
It seems like this is an issue with the hidpi factor chosen on your desktop, and this workaround isn't the right way to fix this issue.